tests/unit/test_spinel_buffer.cpp

```diff
@@ -524,14 +524,14 @@ void TestBuffer(void)
         WriteTestFrame3(ncpBuffer, Spinel::Buffer::kPriorityHigh);
 
         ncpBuffer.InFrameBegin((j % 2) == 0 ? Spinel::Buffer::kPriorityHigh : Spinel::Buffer::kPriorityLow);
-        ncpBuffer.InFrameFeedData(sHelloText, sizeof(sHelloText));
+        IgnoreError(ncpBuffer.InFrameFeedData(sHelloText, sizeof(sHelloText)));
```

Comment:
```suggestion
        SuccessOrQuit(ncpBuffer.InFrameFeedData(sHelloText, sizeof(sHelloText)));
```
Since we are in unittest, is `SuccessOrQuit` better than ignoring?

Reply:
Yes. Will revise accordingly.